examples/nlp/language_modeling/megatron_bart_pretraining.py

```diff
@@ -63,7 +64,9 @@ def main(cfg) -> None:
     if cfg.get('cluster_type', None) == 'BCP':
         plugins.append(TorchElasticEnvironment())
 
-    trainer = Trainer(plugins=plugins, strategy=strategy, **cfg.trainer, callbacks=[ModelSummary(max_depth=3)])
+    trainer = Trainer(
+        plugins=plugins, strategy=strategy, **cfg.trainer, callbacks=[ModelSummary(max_depth=3), CustomProgressBar()]
```

Comment:
Adding the `CustomProgressBar` callback to megatron files that don't use `MegatronTrainerBuilder`